Gopkg.toml

```diff
@@ -24,13 +24,18 @@ ignored = ["github.com/fnproject/fn/cli",
            "github.com/Microsoft/go-winio*",
            "github.com/Azure/go-ansiterm*"]
 
+[prune]
+  go-tests = true
+  unused-packages = true
+  non-go = true
+
 [[constraint]]
   name = "github.com/boltdb/bolt"
   revision = "fa5367d20c994db73282594be0146ab221657943"
 
 [[constraint]]
   name = "github.com/gin-gonic/gin"
-  version = "v1.2"
+  version = "v1.3.0"
```

Comment:
we need 1.3.1 for this bug fix https://github.com/fsouza/go-dockerclient/releases/tag/v1.3.1

Reply:
sorry, wrong dep. moving comment...